udata/core/user/models.py

```diff
@@ -120,9 +122,14 @@ def external_url(self):
         return self.url_for(_external=True)
 
     @property
-    def visible(self):
+    def visible_for_search(self):
         count = self.metrics.get('datasets', 0) + self.metrics.get('reuses', 0)
-        return count > 0
+        return count > 0 and self.active
+
+    @property
+    def visible(self):
+        return self.active or (
+            not current_user.is_anonymous and current_user.sysadmin)
```

Comment:
I am lost...
Do you think that:
1) the code I propose (which uses current_user), is correct. That would be great ;-)
2) whe should check that current_user is defined. In that case, I have not been able to reproduce this situation. Can you give me a scenario to reproduce it ?